src/graph/planner/ngql/GoPlanner.cpp

```diff
@@ -90,11 +90,9 @@ Expression* GoPlanner::loopCondition(uint32_t steps, const std::string& var) {
   return LogicalExpression::makeAnd(pool, step, earlyEnd);
 }
 
-/*
- * extract vid and edge's prop from GN
- * for joinDst & joinInput
- * output colNames {srcProps, edgeProps, kVid, "JOIN_DST_VID"}
- */
+// extract vid and edge's prop from GN
+// for joinDst & joinInput
+// output colNames {srcProps, edgeProps, kVid, "JOIN_DST_VID"}
```

Comment:
Please capitalize the first character and change the verb to third person singular form.
Ref: https://google.github.io/styleguide/cppguide.html#Function_Comments